class UsernameAlreadyExists(Exception):
    pass


class CredentialsException(Exception):
    pass
